be/src/formats/orc/apache-orc/c++/include/orc/Common.hh

```diff
@@ -216,40 +217,34 @@ inline bool compare(T val1, T val2) {
 // Specialization for Decimal
 template <>
```

Comment:
suggest to  put implementation in cpp file, declaration in header file inorder to speed up compilation.

Reply:
let's do this later~